### ISSUE TITLE
Cannot expand list of function

### ISSUE THREAD

**Issue by stackdump** [microsoft/azuredatastudio-postgresql#100]
Attempting to browse functions a PostgreSQL server I'm connected to
I see an error "Failed to expand node"

I'm able to browse other parts of the DB - so it's mostly working for me.

My DB is running in docker: 
```
FROM postgres:11.1
```

From my postgres server logs

DB version
```
psql (10.6 (Ubuntu 10.6-0ubuntu0.18.10.1), server 11.2 (Debian 11.2-1.pgdg90+1))
```
```
2019-03-20 19:16:59.612 UTC [750] ERROR:  column "proisagg" does not exist at character 668
2019-03-20 19:16:59.612 UTC [750] HINT:  Perhaps you meant to reference the column "pr.prolang".
2019-03-20 19:16:59.612 UTC [750] STATEMENT:
        SELECT
            pr.oid,
            pr.proname || '(' || COALESCE(pg_catalog.pg_get_function_identity_arguments(pr.oid), '') || ')' as name,
            lanname,
            pg_get_userbyid(proowner) as funcowner,
            description,
            nsp.nspname AS schema,
            nsp.oid AS schemaoid,
            CASE
            WHEN nsp.nspname like 'pg_%' or nsp.nspname = 'information_schema'
                THEN true
            ELSE false END as is_system
        FROM
            pg_proc pr
        INNER JOIN
            pg_namespace nsp ON pr.pronamespace= nsp.oid
        JOIN
            pg_type typ ON typ.oid=prorettype
        JOIN
            pg_language lng ON lng.oid=prolang
        LEFT OUTER JOIN
            pg_description des ON (des.objoid=pr.oid AND des.classoid='pg_proc'::regclass)
        WHERE
            proisagg = FALSE
            AND typname NOT IN ('trigger', 'event_trigger')
        ORDER BY
            proname;
```

**Comment by akamath46297** [comment]
@stackdump , thanks for reporting the issue. We are investigating this further.

**Comment by nisargrajpara** [comment]
similar issues when connected to greenplum/postgres

**Comment by jonragnarsson** [comment]
Happens on PostgreSQL 10 on debian. Connection to 9.6 on the same server works fine.

**Comment by benny120385** [comment]
same issue here, 
cannot open function on 
localhost PostgreSQL 11.3, compiled by Visual C++ build 1914, 64-bit
but on this server it works:
PostgreSQL 9.5.16 on x86_64-pc-linux-gnu (Ubuntu 9.5.16-1.pgdg14.04+1), compiled by gcc (Ubuntu 4.8.4-2ubuntu1~14.04.4) 4.8.4, 64-bit

**Comment by jcraven-worksight** [comment]
also experiencing this issue on Kubuntu 18.04.

**Comment by sambeckingham** [comment]
Experiencing this also on macOS, PostgreSQL 11.3

**Comment by sambeckingham** [comment]
Still experiencing this in version 1.11.0

**Comment by akamath46297** [comment]
Could you try latest release, it should resolve this issue - https://github.com/microsoft/azuredatastudio-postgresql/releases/tag/v0.2.2

**Comment by jcraven-worksight** [comment]
thank you @akamath46297. I can expand the Functions dropdown in v0.2.2, but I'm unable to view the source of any of the functions?  Double clicking a function name doesn't do anything. Is this intended?

**Comment by akamath46297** [comment]
Fixed with https://github.com/microsoft/pgtoolsservice/pull/219 . 
 https://github.com/microsoft/azuredatastudio-postgresql/releases/tag/v0.2.2

**Comment by juliusbartolome** [comment]
Hello @akamath46297 , I am experiencing this issue with:
OS: macOS Catalina 10.15.2
PostgreSQL: 11.3
ADS: 1.14.1
ADS-PostgreSQL: 0.2.2
pgsqltoolsservice: 1.2.0 (OSX)

I tried installing using the install button in extensions and using the VSIX.
1. When installing using the install button, it was working as expected with the exception of `functions` not expanding.
2. When installing using VSIX I am getting an error in the output window
```
launching server using command /Users/user/.azuredatastudio/extensions/microsoft.azuredatastudio-postgresql-0.2.2/out/pgsqltoolsservice/OSX/v1.2.0/pgsqltoolsservice/pgtoolsservice_main failed.
```
** I tried deleting the `microsoft.azuredatastudio-postgresql-0.2.2` and then reinstalling the plugin using VSIX but has the same result.
